### DISCUSSION TITLE
SpellFlashCore for Shadowlands

### DISCUSSION THREAD

**Discussion by Hekili** [Hekili/hekili#1208] [General]
Hi all,

I recently had an issue ticket come in where SpellFlash was not working properly for default action bars.  This issue was caused by SpellFlashCore no longer being maintained -- the version on Curse was last updated for Battle for Azeroth.  The issue would present itself such that only the first button on your first action bar would ever flash.

**If you want to use SpellFlashCore with the default action bars, you will need to get an updated version of SpellFlashCore.**

This GitHub repository has modernized SpellFlashCore so it will continue to work:  https://github.com/Tga123/SpellFlashCore

![image](https://user-images.githubusercontent.com/9438134/129285055-aba5e71e-bb89-4fea-b37b-4e8b87b34672.png)

Click Code, Download ZIP, and then unpack the ZIP into your `Interface\AddOns` folder.  You will probably need to rename the folder that is created from `SpellFlashCore-main` to `SpellFlashCore`.  After reloading your UI, SpellFlash functionality should be restored.

**Comment by Zebser** [comment]
Thank you for the help

I'm using "Hekili-v9.2.0-1.0.11.zip", uploaded the new SpellFlashCore folder.
Enabled the "SpellFlash" on "Hekili", but still does not work. Missing someting?

![image](https://user-images.githubusercontent.com/46788681/159272740-dc8327ce-9f1b-46e9-8d62-bfe86963b4e1.png)

Thank you.

**Comment by Hekili** [comment]
You don't need to upload anything SpellFlashCore related; it's packaged with this addon.

I'll be posting a fix in a little bit. Thanks!

**Reply by Hekili** [reply]
This is fixed in the new release.

**Reply by Zebser** [reply]
Thank you! @Hekili
